test/lib/specHelperSpec.js

```diff
@@ -95,7 +101,8 @@ describe('Fluxxed up test helpers', function() {
       // Manipulate the DOM:
       rig.fillIn('input', 'new thing')
       rig.clickLink('Submit')
-      rig.toggleCheckbox('#checkbox-input')
+      var element = rig.domNode.find('#checkbox-input')[0]
+      rig.toggleCheckbox(element)
```

Comment:
@andela-tkomolafe Once you make the changes I mentioned above allowing you to just pass the `selector` these two lines can just become `rig.toggleCheckbox('#checkbox-input')`, which is a lot simpler